Contents.m

```diff
@@ -0,0 +1,8 @@
+% MATLAB-bindings for coupling library preCICE
+% Version 2.1.0.1 (R2018b) 14-Oct-2020
+%
+% Files
+%   getVersionInformation - 
```

Comment:
Again: What does this mean? Is there any more information that we could/should provide here?

Reply:
Here the file name was picked up while generating the Contents.m file via the *Report* functionality of MATLAB. 
**Open Question**: Do we provide a full set of `help` instructions where we explain each API function?

Reply:
If it was automatically generated, let's not touch it :smirk: 

A full set of `help` instructions would be nice, but probably too much for this release. I opened an issue on this: https://github.com/precice/matlab-bindings/issues/26.